### ISSUE TITLE
Improve bugfixing of models via easy export functionality?

### ISSUE THREAD

**Issue by llavall** [RakipInitiative/ModelRepository#286]
I have realised that each time an FSKX is failing to run it is giving the person who has to find out why a hard time. One way to solve that is to copy all important model parts out of the FSKX-file (which is not easy at all) in order to run that in the native environment of the model (e.g. Python or R). Since we already have an FSK to R node, 

I was wondering if it is an easy to accomplish option to allow the user easily to simply attach an R node for example and find the ready to run model in there and just run it.

This would allow easy bugfixing, could serve as a test if the FSK result is the same and may open other options for the user.

Best,
Lars

**Comment by miguelalba** [comment]
No. This just creates extra work. What we need is to control properly every error (checked exception) in the runner so that the stack traces are not hidden and a meaningful error is shown in the console. This is something Thomas is already working on.

**Comment by mfilter** [comment]
I still think the suggestion is a valid one. Unfortunately we do not have time for it now, so we must give it a lower prio
